shoop/admin/static_src/base/js/browse-widget.js

```diff
@@ -42,11 +42,13 @@ window.BrowseAPI = (function() {
      * Currently supported options are:
      * * `kind`: kind string (e.g. "product")
      * * `filter`: filter string (kind-dependent)
+     * * `shop`: shop id option for product-list
```

Comment:
This shouldn't be an additional option; please leverage `filter` instead (maybe put JSON there, like `{"shop": 1}`?)